cmd/kube-apiserver/app/options/options.go

```diff
@@ -192,7 +192,7 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 	// TODO (khenidak) change documentation as we move IPv6DualStack feature from ALPHA to BETA
 	fs.StringVar(&s.ServiceClusterIPRanges, "service-cluster-ip-range", s.ServiceClusterIPRanges, ""+
 		"A CIDR notation IP range from which to assign service cluster IPs. This must not "+
-		"overlap with any IP ranges assigned to nodes or pods.")
+		"overlap with any IP ranges assigned to nodes or pods. Currently, the largest supported service subnet is /12 for IPv4 and /108 for IPv6.")
```

Comment:
```suggestion
		"overlap with any IP ranges assigned to nodes or pods. The largest supported service subnet is /12 for IPv4 and /108 for IPv6.")
```
?

“Currently” doesn't really add anything there. It's good to track improvements but docs aren't the best place to track them.

Reply:
@sftim Thank you for the review. If you don't mind, I would like to keep "currently" as I see it being used in other places as well. Also, it suggests to the user that this may not be the case going forward as we do have open issues seeking to address this.

Reply:
I _think_ we should try to remove “Currently” from those other places, based on the style guide for documentation:
https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future

Reply:
Done!